src/providers/storage/cortexiaApi.ts

```diff
@@ -0,0 +1,143 @@
+import { IStorageProvider } from "./storageProviderFactory";
+import { StorageType, IAsset, AssetType, IAssetMetadata, IRegion } from "../../models/applicationState";
+import { AssetService } from "../../services/assetService";
+import apiService, { IImage, IImageWithAction } from "../../services/apiService";
+import { appInfo } from "../../common/appInfo";
+
+export class CortexiaApi implements IStorageProvider {
+    /**
+     * Storage type
+     * @returns - StorageType.Cloud
+     */
+    public storageType: StorageType = StorageType.Cloud;
+
+    /**
+     * @description - Gets images with the provider and returns stringified asset metadata.
+     * @param filePath
+     */
+    public async readText(filePath: string): Promise<any> {
```

Comment:
You return `Promise.resolve(JSON.stringify(...)` so this function always return `Promise<string>`, right?